cypress/integration/components/Select.spec.js

```diff
@@ -163,3 +233,15 @@ describe("Select", () => {
   });
   describe("Knobs", () => {});
 });
+
+function paste({ destinationSelector, pastePayload, pasteType = "text/plain" }) {
+  // https://developer.mozilla.org/en-US/docs/Web/API/Element/paste_event
+  cy.get(destinationSelector).then(($destination) => {
```

Comment:
Is there a reason we're prefixing this with a `$`?
```suggestion
  cy.get(destinationSelector).then((destination) => {
```

Reply:
I took this helper from cypress thread. There is only reason to have `$` just to define that this variable contains element, similar to jQuery style. 
But I will remove, because we do not have such style variables in the project.